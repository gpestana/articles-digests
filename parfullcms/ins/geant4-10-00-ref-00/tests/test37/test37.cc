//
// ********************************************************************
// * License and Disclaimer                                           *
// *                                                                  *
// * The  Geant4 software  is  copyright of the Copyright Holders  of *
// * the Geant4 Collaboration.  It is provided  under  the terms  and *
// * conditions of the Geant4 Software License,  included in the file *
// * LICENSE and available at  http://cern.ch/geant4/license .  These *
// * include a list of copyright holders.                             *
// *                                                                  *
// * Neither the authors of this software system, nor their employing *
// * institutes,nor the agencies providing financial support for this *
// * work  make  any representation or  warranty, express or implied, *
// * regarding  this  software system or assume any liability for its *
// * use.  Please see the license in the file  LICENSE  and URL above *
// * for the full disclaimer and the limitation of liability.         *
// *                                                                  *
// * This  code  implementation is the result of  the  scientific and *
// * technical work of the GEANT4 collaboration.                      *
// * By using,  copying,  modifying or  distributing the software (or *
// * any work based  on the software)  you  agree  to acknowledge its *
// * use  in  resulting  scientific  publications,  and indicate your *
// * acceptance of all terms of the Geant4 Software license.          *
// ********************************************************************
//
#include "G4RunManager.hh"
#include "G4UImanager.hh"
#include "G4UIterminal.hh"
#include "G4UItcsh.hh"
#include "Randomize.hh"

#include "DetectorConstruction.hh"
#include "PhysicsList.hh"
#include "PrimaryGeneratorAction.hh"
#include "RunAction.hh"
#include "EventAction.hh"
#include "SteppingAction.hh"
#include "SteppingVerbose.hh"

//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......

int main(int argc,char** argv) {

  //choose the Random engine
  CLHEP::HepRandom::setTheEngine(new CLHEP::RanecuEngine);

  //my Verbose output class
  G4VSteppingVerbose::SetInstance(new SteppingVerbose);
    
  // Construct the default run manager
  G4RunManager * runManager = new G4RunManager;

  // set mandatory initialization classes
  DetectorConstruction* detector = new DetectorConstruction;
  runManager->SetUserInitialization(detector);
  runManager->SetUserInitialization(new PhysicsList);
  //

  PrimaryGeneratorAction* primary = new PrimaryGeneratorAction();
  runManager->SetUserAction(primary);

   
  // set user action classes
  RunAction*      runAct = new RunAction(detector,primary);
  EventAction*    evtAct = new EventAction();
  SteppingAction* stpAct = new SteppingAction(runAct);
  
  runManager->SetUserAction(runAct);
  runManager->SetUserAction(evtAct);
  runManager->SetUserAction(stpAct);

  // get the pointer to the User Interface manager 
    G4UImanager* UI = G4UImanager::GetUIpointer();  
 
  if (argc==1)   // Define UI terminal for interactive mode  
    { 
      G4UIsession * session = 0;
#ifdef G4UI_USE_TCSH
      session = new G4UIterminal(new G4UItcsh);      
#else
      session = new G4UIterminal();
#endif                      
      session->SessionStart();
      delete session;
    }
  else           // Batch mode
    { 
      G4String command = "/control/execute ";
      G4String fileName = argv[1];
      UI->ApplyCommand(command+fileName);
    }
    
  // job termination

  delete runManager;
  return 0;
}

