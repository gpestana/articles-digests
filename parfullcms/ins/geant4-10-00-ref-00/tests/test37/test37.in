/run/verbose 1
/tracking/verbose 0
#/testem/det/setMat1 G4_Al
#/testem/det/setThick1 826.23935 um
/testem/det/setMat1 G4_Mo
/testem/det/setThick1  203 um
/testem/det/setNbOfLayers1 100 

/testem/phys/addPhysics  standardGS
#/testem/phys/addPhysics  standard
#/testem/phys/addPhysics  emstandard
#/testem/phys/addPhysics  emstandard_opt1
#/testem/phys/addPhysics  emstandard_opt2
#/testem/phys/addPhysics  emstandard_opt3
#/testem/phys/addPhysics  emlivermore
#/testem/phys/addPhysics  empenelope

/gun/particle e- 
/gun/energy 0.5 MeV
#/gun/energy 0.521 MeV
#/gun/energy 0.314 MeV
#/gun/energy 1.033 MeV

# 0 degree
/gun/direction 1.0  0.0  0.0 
# 60 degree
#/gun/direction 0.5  0.0  0.866025404 

#/testem/phys/setCuts 0.7 mm
/testem/phys/setCuts 0.02 mm

/run/initialize

/run/beamOn 10000
