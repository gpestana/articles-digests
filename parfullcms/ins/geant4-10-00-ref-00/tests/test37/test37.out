
        ############################################
        !!! WARNING - FPE detection is activated !!!
        ############################################

*************************************************************
 Geant4 version Name: geant4-10-00-ref-00    (6-December-2013)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

### NIST DataBase for Materials is used
PhysicsList::AddPhysicsList: <standardGS>
PhysicsList::AddPhysicsList: <standardGS> is not defined
### G4EmConfigurator::AddModels n= 0
PhysicsList::SetCuts:CutLength : 1 mm 

phot:   for  gamma    SubType= 12
      LambdaPrime table from 200 keV to 10 TeV in 54 bins 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
       PhotoElectric :  Emin=        0 eV    Emax=       10 TeV   AngularGenSauterGavrila  FluoActive

compt:   for  gamma    SubType= 13
      Lambda table from 100 eV  to 1 MeV in 28 bins, spline: 1
      LambdaPrime table from 1 MeV to 10 TeV in 49 bins 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
       Klein-Nishina :  Emin=        0 eV    Emax=       10 TeV

conv:   for  gamma    SubType= 14
      Lambda table from 1.022 MeV to 10 TeV in 49 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        BetheHeitler :  Emin=        0 eV    Emax=       80 GeV
     BetheHeitlerLPM :  Emin=       80 GeV   Emax=       10 TeV

msc:   for e-    SubType= 10
      RangeFactor= 0.04, stepLimitType: 1, latDisplacement: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=      100 MeV  Table with 42 bins Emin=    100 eV    Emax=    100 MeV
        WentzelVIUni :  Emin=      100 MeV   Emax=       10 TeV  Table with 35 bins Emin=    100 MeV   Emax=     10 TeV

eIoni:   for  e-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        MollerBhabha :  Emin=        0 eV    Emax=       10 TeV

eBrem:   for  e-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      LPM flag: 1 for E > 1 GeV
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
             eBremSB :  Emin=        0 eV    Emax=        1 GeV   DipBustGen
            eBremLPM :  Emin=        1 GeV   Emax=       10 TeV   DipBustGen

CoulombScat:   for  e-    SubType= 1
      Lambda table from 100 MeV to 10 TeV in 35 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=      100 MeV   Emax=       10 TeV

msc:   for e+    SubType= 10
      RangeFactor= 0.04, stepLimitType: 1, latDisplacement: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=      100 MeV  Table with 42 bins Emin=    100 eV    Emax=    100 MeV
        WentzelVIUni :  Emin=      100 MeV   Emax=       10 TeV  Table with 35 bins Emin=    100 MeV   Emax=     10 TeV

eIoni:   for  e+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        MollerBhabha :  Emin=        0 eV    Emax=       10 TeV

eBrem:   for  e+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      LPM flag: 1 for E > 1 GeV
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
             eBremSB :  Emin=        0 eV    Emax=        1 GeV   DipBustGen
            eBremLPM :  Emin=        1 GeV   Emax=       10 TeV   DipBustGen

annihil:   for  e+    SubType= 5
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            eplus2gg :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  e+    SubType= 1
      Lambda table from 100 MeV to 10 TeV in 35 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=      100 MeV   Emax=       10 TeV

msc:   for proton    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  proton    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=        2 MeV
          BetheBloch :  Emin=        2 MeV   Emax=       10 TeV

hBrems:   for  proton    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  proton    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 13x1001 from 7.50618 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for GenericIon    SubType= 10
      RangeFactor= 0.2, stepLimitType: 0, latDisplacement: 0
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=       10 TeV

ionIoni:   for  GenericIon    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.01, dRoverRange= 0.1, integral: 1, fluct: 1, linLossLimit= 0.02
      Stopping Power data for 17 ion/material pairs 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            BraggIon :  Emin=        0 eV    Emax=        2 MeV
          BetheBloch :  Emin=        2 MeV   Emax=       10 TeV

msc:   for alpha    SubType= 10
      RangeFactor= 0.2, stepLimitType: 0, latDisplacement: 0
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

ionIoni:   for  alpha    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.01, dRoverRange= 0.1, integral: 1, fluct: 1, linLossLimit= 0.02
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            BraggIon :  Emin=        0 eV    Emax=   7.9452 MeV
          BetheBloch :  Emin=   7.9452 MeV   Emax=       10 TeV

msc:   for anti_proton    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  anti_proton    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=        2 MeV
          BetheBloch :  Emin=        2 MeV   Emax=       10 TeV

hBrems:   for  anti_proton    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  anti_proton    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 13x1001 from 7.50618 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for kaon+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  kaon+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=  1.05231 MeV
          BetheBloch :  Emin=  1.05231 MeV   Emax=       10 TeV

hBrems:   for  kaon+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  kaon+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 14x1001 from 3.94942 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for kaon-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  kaon-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=  1.05231 MeV
          BetheBloch :  Emin=  1.05231 MeV   Emax=       10 TeV

hBrems:   for  kaon-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  kaon-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 14x1001 from 3.94942 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for mu+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

muIoni:   for  mu+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=      200 keV
          BetheBloch :  Emin=      200 keV   Emax=        1 GeV
        MuBetheBloch :  Emin=        1 GeV   Emax=       10 TeV

muBrems:   for  mu+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
              MuBrem :  Emin=        0 eV    Emax=       10 TeV

muPairProd:   for  mu+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 17x1001 from 1 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
          muPairProd :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  mu+    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 38 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=        0 eV    Emax=       10 TeV

msc:   for mu-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

muIoni:   for  mu-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=      200 keV
          BetheBloch :  Emin=      200 keV   Emax=        1 GeV
        MuBetheBloch :  Emin=        1 GeV   Emax=       10 TeV

muBrems:   for  mu-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
              MuBrem :  Emin=        0 eV    Emax=       10 TeV

muPairProd:   for  mu-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 17x1001 from 1 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
          muPairProd :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  mu-    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 38 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=        0 eV    Emax=       10 TeV

msc:   for pi+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  pi+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=  297.505 keV
          BetheBloch :  Emin=  297.505 keV   Emax=       10 TeV

hBrems:   for  pi+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  pi+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 16x1001 from 1.11656 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for pi-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 77 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  pi-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=  297.505 keV
          BetheBloch :  Emin=  297.505 keV   Emax=       10 TeV

hBrems:   for  pi-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV

hPairProd:   for  pi-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 77 bins
      Lambda tables from threshold to 10 TeV in 77 bins, spline: 1
      Sampling table 16x1001 from 1.11656 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

========= Table of registered couples ==============================

Index : 0     used in the geometry : Yes
 Material : G4_Galactic
 Range cuts        :  gamma  20 um     e-  20 um     e+  20 um  proton 1 mm 
 Energy thresholds :  gamma  990 eV     e-  990 eV     e+  990 eV  proton 100 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

Index : 1     used in the geometry : Yes
 Material : G4_Mo
 Range cuts        :  gamma  20 um     e-  20 um     e+  20 um  proton 1 mm 
 Energy thresholds :  gamma  4.52077 keV    e-  94.2689 keV    e+  93.0104 keV proton 100 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

====================================================================

### Run 0 starts.
### Run 0 start E(MeV)= 0.5   diff= -0.533
### Event No 0
Run terminated.
Run Summary
  Number of events processed : 10000
  User=7s Real=7.12s Sys=0s
Thicknesses(mm)= 1.57423  1  1  
Bins(R/R0)=      0.00738313  0  0  
 ----------------------------------------------------------
 ----------------  RUN SUMMARY ----------------------------
 ----------------------------------------------------------
       Mean number of steps 123.618
             Medium 1 ==>   G4_Mo
	 FMR (z/r0)    ||       J  
	   0.00738313	          2.57441
	   0.0147663	          2.98701
	   0.0221494	          3.31975
	   0.0295325	          3.78083
	   0.0369157	          4.10358
	   0.0442988	          4.48162
	   0.0516819	          4.74705
	   0.0590651	          4.72877
	   0.0664482	          5.00961
	   0.0738313	          5.14098
	   0.0812144	          5.15802
	   0.0885976	          5.20023
	   0.0959807	          5.29521
	   0.103364	          5.31987
	   0.110747	          5.23554
	   0.11813	          5.11525
	   0.125513	          4.95385
	   0.132896	          4.96556
	   0.14028	          4.85021
	   0.147663	          4.83512
	   0.155046	          4.64289
	   0.162429	          4.69667
	   0.169812	          4.51406
	   0.177195	          4.4057
	   0.184578	          4.29371
	   0.191961	          4.15604
	   0.199345	          3.84112
	   0.206728	          3.82156
	   0.214111	          3.75253
	   0.221494	          3.51119
	   0.228877	          3.3411
	   0.23626	          3.16154
	   0.243643	          3.04426
	   0.251026	          2.75229
	   0.25841	          2.71795
	   0.265793	          2.72696
	   0.273176	          2.46782
	   0.280559	          2.41351
	   0.287942	          2.2582
	   0.295325	          2.14332
	   0.302708	          2.02079
	   0.310092	          1.97001
	   0.317475	          1.868
	   0.324858	          1.71682
	   0.332241	          1.65769
	   0.339624	          1.46742
	   0.347007	          1.28805
	   0.35439	          1.22265
	   0.361773	          1.15275
	   0.369157	          1.06079
	   0.37654	          1.02659
	   0.383923	          0.923122
	   0.391306	          0.890701
	   0.398689	          0.776754
	   0.406072	          0.749582
	   0.413455	          0.718284
	   0.420839	          0.593372
	   0.428222	          0.537442
	   0.435605	          0.556957
	   0.442988	          0.446746
	   0.450371	          0.429649
	   0.457754	          0.379608
	   0.465137	          0.346457
	   0.47252	          0.324296
	   0.479904	          0.300827
	   0.487287	          0.27199
	   0.49467	          0.256176
	   0.502053	          0.191642
	   0.509436	          0.174026
	   0.516819	          0.162782
	   0.524202	          0.140541
	   0.531585	          0.0965664
	   0.538969	          0.096483
	   0.546352	          0.0828917
	   0.553735	          0.0946201
	   0.561118	          0.0754923
	   0.568501	          0.0560394
	   0.575884	          0.0338348
	   0.583267	          0.0322573
	   0.590651	          0.0234935
	   0.598034	          0.0291373
	   0.605417	          0.0242262
	   0.6128	          0.0178159
	   0.620183	          0.0184904
	   0.627566	          0.0159569
	   0.634949	          0.0180263
	   0.642332	          0.0155856
	   0.649716	          0.0093662
	   0.657099	          0.00855776
	   0.664482	          0.00690413
	   0.671865	          0.00716125
	   0.679248	          0.00498537
	   0.686631	          0.0102517
	   0.694014	          0.00799691
	   0.701398	          0.00568953
	   0.708781	          0.00572611
	   0.716164	          0.00596328
	   0.723547	          0.00288892
	   0.73093	          0.00100126
	   0.738313	          0.00115806

 Deposit energy (MeV) = 0.387746
 ----------------------------------------------------------
 ----------------------------------------------------------
 ----------------------------------------------------------
 Total Deposit energy (MeV) = 0.387746
 ----------------------------------------------------------
 ----------------  END OF RUN SUMMARY ---------------------
 ----------------------------------------------------------
G4 kernel has come to Quit state.
================== Deleting memory pools ===================
Number of memory pools allocated: 10 of which, static: 0
Dynamic pools deleted: 10 / Total memory freed: 0.25 Mb
============================================================
RunManagerKernel is deleted. Good bye :)
